app/validators/name_lenght_validator.rb

```diff
@@ -0,0 +1,9 @@
+class NameLenghtValidator < ActiveModel::EachValidator
```

Comment:
It this is a part of validators you should also have in model usage of this validator.